R/sptidy.R

```diff
@@ -32,12 +32,29 @@ tidy_lr <- function() {
 tidy_kmeans <- function() {
 }
 
-#' Title
+#' Augmented Output for Tidymodel's Linear Regression
 #'
-#' @return
+#' Adds two columns to the original data of the scikit learn's linear regression model. This includes predictions and residuals.
+#'
+#' @params model _lm
+#' @params X data.frame
+#' @params y data.frame
```

Comment:
```suggestion
#' 
```
Hi Heidi! I think for tidymodels we do not need a y dataframe to fit the model.

Reply:
Good to know! Thanks!